py/client/pydeephaven/session.py

```diff
@@ -99,23 +113,29 @@ class Session:
         is_alive (bool): check if the session is still alive (may refresh the session)
     """
 
-    def __init__(self, host: str = None,
-                 port: int = None,
-                 auth_type: str = "Anonymous",
-                 auth_token: str = "",
-                 never_timeout: bool = True,
-                 session_type: str = 'python',
-                 use_tls: bool = False,
-                 tls_root_certs: bytes = None,
-                 client_cert_chain: bytes = None,
-                 client_private_key: bytes = None,
-                 client_opts: List[Tuple[str, Union[int, str]]] = None,
-                 extra_headers: Dict[bytes, bytes] = None):
+    host: str
+    port: int
+
+    def __init__(
+        self,
+        host: Optional[str] = None,
+        port: Optional[int] = None,
+        auth_type: str = "Anonymous",
+        auth_token: str = "",
+        never_timeout: bool = True,
+        session_type: str = "python",
+        use_tls: bool = False,
+        tls_root_certs: Optional[bytes] = None,
+        client_cert_chain: Optional[bytes] = None,
+        client_private_key: Optional[bytes] = None,
+        client_opts: Optional[list[tuple[str, Union[int, str]]]] = None,
+        extra_headers: Optional[dict[bytes, bytes]] = None,
```

Comment:
Should any of these other properties be documented above with `host` and `port`?